dangerzone/container.py

```diff
@@ -29,6 +27,106 @@
 container_name = "dangerzone.rocks/dangerzone"
 
 
+class NoContainerTechException(Exception):
+    def __init__(self, container_tech: str) -> None:
+        super().__init__(f"{container_tech} is not installed")
+
+
+def get_runtime_name() -> str:
+    if platform.system() == "Linux":
+        runtime_name = "podman"
+    else:
+        # Windows, Darwin, and unknown use docker for now, dangerzone-vm eventually
+        runtime_name = "docker"
+    return runtime_name
+
+
+def get_runtime() -> str:
+    container_tech = get_runtime_name()
+    runtime = shutil.which(container_tech)
+    if runtime is None:
+        raise NoContainerTechException(container_tech)
+    return runtime
+
+
+def install() -> Optional[bool]:
```

Comment:
I would either return `bool`, or keep the `Optional` but raise an exception (to be caught) upon failure to install.

Reply:
Changed to bool. I have an issue already for handling container installation failure https://github.com/freedomofpress/dangerzone/issues/193.